oleh-burger-app/src/components/Main/Prices/Ingredient/Ingredient.js

```diff
@@ -0,0 +1,11 @@
+import './Ingredient.css'
+
+function Ingredient ({ingredients}) {
+    return (
+        <ul className='prices_ingredients'>
+            {ingredients.map((element) => <li className='ingredient_icon' key={`Ingredient-icon #${ingredients.indexOf(element)}`}>{element.name}: {element.price}</li>)}
```

Comment:
Try to avoid indexes as key values

https://robinpokorny.com/blog/index-as-a-key-is-an-anti-pattern/